database/index.js

```diff
@@ -19,7 +19,7 @@ connection.connect((err) => {
 });
 
 const getData = (callback) => {
-  const queryStr = `SELECT reviews.*, properties.host_name FROM reviews, properties WHERE reviews.hostId = ${5} AND properties.id = ${5}`;
+  const queryStr = `SELECT reviews.*, properties.host_name, properties.host_photo, properties.host_text FROM reviews, properties WHERE reviews.hostId = ${5} AND properties.id = ${5}`;
```

Comment:
hostId and properties.id are hard-coded with 5 here?

Reply:
I'm just hard coding with a single id for development reasons. When we have our proxy server up and all the components combined, then I can render whatever data is for the room id the customer is looking at.